src/channels/whatsapp/streaming_integration_enhanced.py

```
@@ -0,0 +1,505 @@
+            
+            # Log error in trace context
+            if trace_context:
+                trace_context.log_streaming_error(e, "streaming_general")
+            
+            # Try to send error message to user
+            try:
+                await self.message_queue_handler.send_message_with_delay(
+                    recipient, "Sorry, I encountered a technical error. Please try again."
+                )
+            except:
+                pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)